pkg/init/Dockerfile

```diff
@@ -1,17 +1,28 @@
-FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 AS build
-RUN apk add --no-cache --initdb alpine-baselayout make gcc musl-dev
+FROM linuxkit/alpine:0fd732eb9e99c4db0953ae8de23d95de340ab847 AS build
+RUN apk add --no-cache --initdb alpine-baselayout make gcc musl-dev git linux-headers
 
-ADD usermode-helper.c .
-RUN make usermode-helper
+ADD usermode-helper.c ./
+RUN LDFLAGS=-static CFLAGS=-Werror make usermode-helper
 
 RUN apk add --no-cache go musl-dev
 ENV GOPATH=/go PATH=$PATH:/go/bin
 
-COPY init.go /go/src/init/
-COPY vendor /go/src/init/vendor/
-RUN go-compile.sh /go/src/init/
+COPY cmd /go/src/cmd
+RUN go-compile.sh /go/src/cmd/init
 
-FROM linuxkit/alpine:87a0cd10449d72f374f950004467737dbf440630 AS mirror
+# checkout containerd for vendoring
```

Comment:
Not completely related to this PR, but I wonder if we should just do the clone in the alpine base andthen consumers can just use it (or copy it, or symlink it)